pipeline/load.py

```diff
@@ -24,45 +22,44 @@ def get_db_connection():
         return {'error': 'Unable to connect to the database.'}
 
 
-def split_df(df: pd.DataFrame) -> tuple:
-    pass
-
-
 def upload_stories(df: pd.DataFrame) -> None:
-    """Uploads dataframe to stories table. If value already """
-    # topic_id
+    """Gets stories and records data from dataframe. Uploads it to database tables."""
 
-    query = """
+    story_query = """
             INSERT INTO stories
```

Comment:
Need to account for topic_id

Reply:
Yep, just going to pull this review once complete and add it in a new commit. 👍